src/Microsoft.Health.Dicom.Functions/Durable/RetryOptionsTemplate.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Dicom.Functions.Durable
+{
+    internal sealed class RetryOptionsTemplate
+    {
+        public TimeSpan FirstRetryInterval { get; set; } = TimeSpan.Zero;
+
+        public TimeSpan MaxRetryInterval { get; set; } = TimeSpan.FromDays(6.0);
```

Comment:
would 6 days too big?
Imaging the activity is not executed until 6 days later...

Reply:
I should add a comment. This is their internal max lol and the default max retry interval. This is to serve as a ceiling for the retry delay. Consider the following piecewise function, based on my reading of their [documentation](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-error-handling?tabs=csharp#automatic-retry-on-failure)

Let D(i) be the delay before the i-th retry where 1 <= i <= *MaxNumberOfAttempts* and *MaxNumberOfAttempts* > 1. Note that if *MaxNumberOfAttempts* is 1, then there is no retrying the operation.

```
       { MAX(MaxRetryInterval, FirstRetryInterval), if i == 1
D(i) = {
       { MAX(MaxRetryInterval, FirstRetryInterval * BackoffCoefficient * (i - 1)), if i > 1
```